api/v1beta1/event.proto

```diff
@@ -9,9 +9,597 @@ import "google/protobuf/wrappers.proto";
+    bpf = 322;
+    execveat = 323;
+    userfaultfd = 324;
+    membarrier = 325;
+    mlock2 = 326;
+    copy_file_range = 327;
+    preadv2 = 328;
+    pwritev2 = 329;
+    pkey_mprotect = 330;
+    pkey_alloc = 331;
+    pkey_free = 332;
+    statx = 333;
+    io_pgetevents = 334;
+    rseq = 335;
+    pidfd_send_signal = 336;
+    io_uring_setup = 337;
+    io_uring_enter = 338;
+    io_uring_register = 339;
+    open_tree = 340;
+    move_mount = 341;
+    fsopen = 342;
+    fsconfig = 343;
+    fsmount = 344;
+    fspick = 345;
+    pidfd_open = 346;
+    clone3 = 347;
+    close_range = 348;
+    openat2 = 349;
+    pidfd_getfd = 350;
+    faccessat2 = 351;
+    process_madvise = 352;
+    epoll_pwait2 = 353;
+    mount_setattr = 354;
+    quotactl_fd = 355;
+    landlock_create_ruleset = 356;
+    landlock_add_rule = 357;
+    landlock_restrict_self = 358;
+    memfd_secret = 359;
+    process_mrelease = 360;
+    waitpid = 361;
+    oldfstat = 362;
+    break = 363;
+    oldstat = 364;
+    umount = 365;
+    stime = 366;
+    stty = 367;
+    gtty = 368;
+    nice = 369;
+    ftime = 370;
+    prof = 371;
+    signal = 372;
+    lock = 373;
+    mpx = 374;
+    ulimit = 375;
+    oldolduname = 376;
+    sigaction = 377;
+    sgetmask = 378;
+    ssetmask = 379;
+    sigsuspend = 380;
+    sigpending = 381;
+    oldlstat = 382;
+    readdir = 383;
+    profil = 384;
+    socketcall = 385;
+    olduname = 386;
+    idle = 387;
+    vm86old = 388;
+    ipc = 389;
+    sigreturn = 390;
+    sigprocmask = 391;
+    bdflush = 392;
+    afs_syscall = 393;
+    llseek = 394;
+    old_select = 395;
+    vm86 = 396;
+    old_getrlimit = 397;
+    mmap2 = 398;
+    truncate64 = 399;
+    ftruncate64 = 400;
+    stat64 = 401;
+    lstat64 = 402;
+    fstat64 = 403;
+    lchown16 = 404;
+    getuid16 = 405;
+    getgid16 = 406;
+    geteuid16 = 407;
+    getegid16 = 408;
+    setreuid16 = 409;
+    setregid16 = 410;
+    getgroups16 = 411;
+    setgroups16 = 412;
+    fchown16 = 413;
+    setresuid16 = 414;
+    getresuid16 = 415;
+    setresgid16 = 416;
+    getresgid16 = 417;
+    chown16 = 418;
+    setuid16 = 419;
+    setgid16 = 420;
+    setfsuid16 = 421;
+    setfsgid16 = 422;
+    fcntl64 = 423;
+    sendfile32 = 424;
+    statfs64 = 425;
+    fstatfs64 = 426;
+    fadvise64_64 = 427;
+    clock_gettime32 = 428;
+    clock_settime32 = 429;
+    clock_adjtime64 = 430;
+    clock_getres_time32 = 431;
+    clock_nanosleep_time32 = 432;
+    timer_gettime32 = 433;
+    timer_settime32 = 434;
+    timerfd_gettime32 = 435;
+    timerfd_settime32 = 436;
+    utimensat_time32 = 437;
+    pselect6_time32 = 438;
+    ppoll_time32 = 439;
+    io_pgetevents_time32 = 440;
+    recvmmsg_time32 = 441;
+    mq_timedsend_time32 = 442;
+    mq_timedreceive_time32 = 443;
+    rt_sigtimedwait_time32 = 444;
+    futex_time32 = 445;
+    sched_rr_get_interval_time32 = 446;
+
+    // Common events (used by all architectures).
+    net_packet_base = 700;
```

Comment:
We previously tried to make event ids "packed" so 700 was chosen as the start of common events.
Since these values now become part of the API, I think it makes more sense to start the count from 1000.